packages/ui/src/providers/catalog.provider.tsx

```diff
@@ -10,41 +10,31 @@ export const CatalogContext = createContext<typeof CamelCatalogService>(CamelCat
 /**
  * Loader for the components catalog.
  */
-export const CatalogLoaderProvider: FunctionComponent<PropsWithChildren> = (props) => {
+export const CatalogLoaderProvider: FunctionComponent<PropsWithChildren<{ catalogUrl: string }>> = (props) => {
   const [isLoading, setIsLoading] = useState(true);
 
   useEffect(() => {
-    fetch(`.${CatalogSchemaLoader.DEFAULT_CATALOG_PATH}/index.json`)
+    fetch(`${props.catalogUrl}/index.json`)
       .then((response) => response.json())
       .then(async (catalogIndex: CamelCatalogIndex) => {
         const camelComponentsFiles = CatalogSchemaLoader.fetchFile<ComponentsCatalog[CatalogKind.Component]>(
-          catalogIndex.catalogs.components.file,
+          `${props.catalogUrl}/${catalogIndex.catalogs.components.file}`,
         );
         const camelProcessorsFiles = CatalogSchemaLoader.fetchFile<ComponentsCatalog[CatalogKind.Processor]>(
-          catalogIndex.catalogs.models.file,
-        );
-        const camelLanguagesFiles = CatalogSchemaLoader.fetchFile<ComponentsCatalog[CatalogKind.Language]>(
-          catalogIndex.catalogs.languages.file,
-        );
-        const camelDataformatsFiles = CatalogSchemaLoader.fetchFile<ComponentsCatalog[CatalogKind.Dataformat]>(
-          catalogIndex.catalogs.dataformats.file,
```

Comment:
I'm not sure why the languages and dataformats loading was removed

Reply:
Oh, this is a mistake, thanks for catching this @tplevko , this seems like a bad rebase. I'll put them back right away.

Reply:
I restored it back, once again, thanks @tplevko for catching this up